packages/cli/src/ui/hooks/slashCommandProcessor.ts

```diff
@@ -153,6 +154,41 @@ export const useSlashCommandProcessor = (
     [addItem],
   );
 
+  const reloadMcpAction = useCallback(async () => {
+    if (!config) {
+      addMessage({
+        type: MessageType.ERROR,
+        content: 'Configuration not available.',
+        timestamp: new Date(),
+      });
+      return;
+    }
+    addMessage({
+      type: MessageType.INFO,
+      content: 'Reloading MCP servers...',
```

Comment:
nit: use refresh over reload..

```suggestion
      content: 'Refreshing MCP servers...',
```